### PR TITLE
Installation adjustments

### PR DESCRIPTION
**Pull request by woodseowl**
While doing the setup of a potential version 0.1 release on https://laravel-demo-test.hosting.cornell.edu (see https://github.com/CU-CommunityApps/CD-laravel-demo/pull/2), I encountered several things that could help someone installing a site based on the Starter Kit on Media3. I've tried addressing these in configuration and documentation.

**This PR does the following**

- Adds installation configuration of `.env.example` and `composer.json`
- Adds detailed installation and configuration usage directions
- Adds installation question to customize project description
- Adds installation of modified `/public/.htaccess` to address Shibboleth configuration

**To Review:**

- [ ] Follow the README.md "Usage" directions (but don't do the GitHub commands) and confirm that you see the Starter Kit demo site when you have finished
- [ ] Read through the "Deploying a site" section of the README.md and see if you have questions or improvements

### CODE REVIEW

**Review by inaydich — APPROVED** [review]
I am approving. Thanks for putting so much work into it. I would love to test it next week.

File: README.md [hunk 1]
```diff
@@ -13,17 +13,83 @@ A Cornell University CIT Custom Development starter kit package for Laravel.
...
+    @include('cwd-framework-index.blade.php')
+    ```
+
+## Deploying a site
+Once a Media3 site has been created, you have confirmed you can reach the default site via a web browser, and you have access to the site login by command line, the code can be deployed.
+
+You will likely need to map the `php` command to the correct version by editing `~/.bashrc` to include this alias (for this to take effect, run `source ~/.bashrc` or just log in again):
+```shell
+# User specific aliases and functions
+alias php="/opt/cpanel/ea-php81/root/usr/bin/php"
```
Comment (richmarisa) [inline README.md:61]: I've been using alias php="/usr/local/bin/ea-php81" on all of my sites

Comment (woodseowl) [reply README.md]: Thanks, updated!

**Comment by woodseowl** [comment]
Closes #40

**Pull request merged**